Source/Diagnostics/FlushFormats/FlushFormatPlotfile.cpp

```diff
@@ -304,15 +304,13 @@ FlushFormatPlotfile::WriteParticles(const std::string& dir,
         }
 #endif
 
-        // Convert momentum to SI
-        pc->ConvertUnits(ConvertDirection::WarpX_to_SI);
-
         RandomFilter const random_filter(particle_diags[i].m_do_random_filter,
                                          particle_diags[i].m_random_fraction);
         UniformFilter const uniform_filter(particle_diags[i].m_do_uniform_filter,
                                            particle_diags[i].m_uniform_stride);
         ParserFilter const parser_filter(particle_diags[i].m_do_parser_filter,
-                                         getParser(particle_diags[i].m_particle_filter_parser));
+                                         getParser(particle_diags[i].m_particle_filter_parser),
```

Comment:
If we don't call `pc->ConvertUnits(ConvertDirection::WarpX_to_SI);` before this, which unit does the momentum have then at that point?